tools/cli/help.txt

```diff
@@ -474,11 +463,11 @@ Usage: meteor deploy <site> [--settings settings.json] [--debug] [--delete]
 
 Deploys the project in your current directory to Meteor's servers.
 
-You can deploy to any available name under 'meteor.com'
+You can deploy to any available name under 'meteorapp.com'
 without any additional configuration, for example,
-'myapp.meteor.com'. If you deploy to a custom domain, such as
+'myapp.meteorapp.com'. If you deploy to a custom domain, such as
 'myapp.mydomain.com', then you'll also need to configure your domain's
-DNS records. See the Meteor docs for details.
+DNS records. See the Meteor / Galaxy docs for details.
```

Comment:
Could we add a link to those docs while we're at it?

Reply:
Good call 👍